R/run_stress_test.R

```diff
@@ -26,7 +26,7 @@
 #'   excluded_companies.csv shall be excluded.
 #' @return NULL
 #' @export
-run_stress_test <- function(asset_type,
+run_stress_test <- function(asset_type = c("bonds", "equity", "loans"),
```

Comment:
while nicer from a design perspective I wonder if this is actually helpful from a user perspective. I see the danger that users do no understand that they have to explicitly select an input type, meaning their first interaction with the code would be a confusing error/warning, whereas if no default is provided it is obvious for the user what to do

Reply:
Great you think of usability. With that in mind again I would think of 1 interface per option -- which reminds me of reading about affordances in The design of eveyday things. A quick google search throws me this:

> Affordance is the relationship between what something looks like and how it's used. For designers, it means that as soon as someone sees something, they have to know how to use it. For example, a mug has high affordance: it's easy to figure out intuitively how to use it.

Reply:
Great, as long as the interface is simply an interface to calling run_stress_test with the correct asset_type  as commented in more detail above let us go that way :)

Reply:
alternative, depening on how you plan to provide the interface for the sensitivity analysis, we could also place the distinguiation between the 3 types there?